pkg/runit/servicebuilder.go

```diff
@@ -124,7 +124,7 @@ func (s ServiceTemplate) logScript() ([]byte, error) {
 	ret := fmt.Sprintf(`#!/usr/bin/ruby
 require 'yaml'
 sleep %d
-exec *YAML.load(DATA.read)
```

Comment:
i wonder if that first part should be `filepath.Basename(DefaultChpst)` or something like that